exercises/twelve-days/canonical-data.json

```diff
@@ -0,0 +1,102 @@
+{
+  "exercise": "twelve-days",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "first day a partridge in a pear tree",
+      "property": "verse",
+      "input": 1,
+      "expected": "On the first day of Christmas my true love gave to me, a Partridge in a Pear Tree.\n"
```

Comment:
I believe the suggestion is that this line should not have the `\n` at the end.

Reply:
Correct

Reply:
Removed trailing newline for single verse tests